server/service/src/sync/translations/item.rs

```diff
@@ -53,7 +58,7 @@ impl SyncTranslation for ItemTranslation {
             code: data.code,
             unit_id: None,
             r#type: to_item_type(data.type_of),
-            legacy_record: sync_record.data.clone(),
+            legacy_record: ordered_simple_json(&sync_record.data)?,
```

Comment:
Yep, it's coming unordered from 4d, this will eventually be replaced with normal fields on item, but for now, to make sure tests pass they need to be ordered (canonical)